cpp/oneapi/dal/algo/svm/backend/gpu/infer_kernel_dpc.cpp

```diff
@@ -90,13 +90,13 @@ static infer_result call_daal_kernel(const context_gpu& ctx,
     auto arr_label = array<Float>::empty(queue, row_count * 1);
     auto event     = queue.submit([&](sycl::handler& cgh) {
         arr_label.need_mutable_data(queue);
```

Comment:
Remove this line. `arr_lable` already has mutable data, because it's allocated on our side, not by user.

Reply:
Thanks! Fixed